java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateJobs.java

```diff
@@ -78,12 +85,23 @@ public CreateJobs(ObjectFactory objectFactory,
                       StateStoreProvider stateStoreProvider,
                       JobSender jobSender,
                       CompactionJobStatusStore jobStatusStore) {
+        this(objectFactory, instanceProperties, tablePropertiesProvider, stateStoreProvider, jobSender, jobStatusStore, false);
+    }
+
+    public CreateJobs(ObjectFactory objectFactory,
+                      InstanceProperties instanceProperties,
+                      TablePropertiesProvider tablePropertiesProvider,
+                      StateStoreProvider stateStoreProvider,
+                      JobSender jobSender,
+                      CompactionJobStatusStore jobStatusStore,
+                      boolean forceCreateJobs) {
```

Comment:
Could we use a static constructor to set this flag? It's a bit difficult to notice that's what's being set from looking at the call to this constructor.